.rubocop.yml

```diff
@@ -28,6 +39,11 @@ Style/Next:
 Style/NonNilCheck:
   Enabled: false
 
+# This cop isn't compatible with Ruby < 2.0
+Style/SymbolArray:
+  Exclude:
+    - "Rakefile"
```

Comment:
Why ignore the Rakefile? I thought that since it is ruby, it should be tested. I'm not sure what I was supposed to get from looking at that Travis link.